src/Lib/App/Env.hs

```diff
@@ -20,3 +21,7 @@ data AppEnv = AppEnv
 newtype Session = Session {
   isLoggedIn :: Bool
 }
+
+newtype JwtSecret = JwtSecret {
+  unJwtSecret :: Text
```

Comment:
`jwtSecret` field in `AppEnv` should have type `JwtSecret` and not `Text`. Also, you need to move `JwtSecret` type into `Lib.Core.Jwt` module.